src/features/content-explorer/content-explorer/ContentExplorer.scss

```
@@ -72,22 +72,50 @@
         }
     }
 
+    .bdl-ContentExplorer-subheader {
+        display: flex;
+        align-items: center;
+
+        .checkbox-container,
+        .toggle-container {
+            margin: 0;
+        }
+
+        .content-explorer-select-all-checkbox {
+            margin-right: 4.45 * $bdl-grid-unit;
+            margin-left: auto;
+        }
+    }
+
     .content-explorer-select-all-container {
-        display: block;
-        height: 8 * $bdl-grid-unit;
+        display: flex;
+        align-items: center;
+        width: 100%;
+        height: 10 * $bdl-grid-unit;
 
         .content-explorer-select-all-items-counter {
             margin-left: 2px;
         }
 
         .content-explorer-select-all-checkbox-label {
-            float: right;
-            margin-right: 42px;
+            display: flex;
+            flex: 1;
+            justify-content: flex-end;
```

I think I see what's happening and I think this should solve it:

set `margin-left: auto;` here instead of these flex styles
```suggestion
            margin-left: auto;
```

then for `.content-explorer-select-all-checkbox` on line 84, try this:
```
.content-explorer-select-all-checkbox {
    margin-right: ...

    &:first-child {
        margin-left: auto;
    }
}
```

---

Yes this also works